isamples_metadata/Transformer.py

```diff
@@ -1,5 +1,83 @@
 from abc import ABC, abstractmethod
 import typing
+from enum import Enum, auto
+
+
+class _ControlledVocabularyCategory(Enum):
+    """Enum subclass for controlled vocbularies -- __repr__ overridden to print the quoted value"""
+
+    def __repr__(self):
+        return "'%s'" % (self.value)
+
+
+class ContextCategory(_ControlledVocabularyCategory):
+    """Top level context, based on the kind of feature sampled, from the SampledFeature vocabulary.  Specific identification of the sampled feature of interest is done through the SamplingEvent/Feature of Interest property."""
+
+    ATMOSPHERE = ("Atmosphere",)
```

Comment:
@datadavev -- Black has chosen this strange combo of comma and parentheses.  Unsure if this is a bug with the way I'm doing it, or a bug in their formatter, or actually correct Python for reasons I don't fully understand.  I have confirmed that doing it this way yields the correct output dictionary for the purposes of transformation.

Reply:
This is what it looks like locally when I run it (as we'd hope):

```
 'hasContextCategory': ['Atmosphere', 'Building'],
 'hasMaterialCategory': ['Fluid material'],
 'hasSpecimenCategory': ['Other solid object'],
```

Reply:
That seems really odd. I read that as an enum of tuples, each containing a single string value. I don't think that is what is intended...

Reply:
More generally, placing these values in code is OK for now, but will be challenging to maintain as the lists evolve. 

We will need to refactor to instead load these controlled lists from a database or data file (perhaps even a json document).

Reply:
Got it, thanks for the confirmation.  The values already exist in a JSON document in the repo, so we could load those up if we needed to display a list of them somewhere.  

The problem we're trying to solve for the purposes of the transformer is that we need to map to a specific String value, which is what the enums are good for.

Maybe I'll just keep these as plain old lists of Strings and let the transformers deal with them that way.  Seems like it might be preferable.